### ISSUE TITLE
Meta should not produce precision 131089 for PostgreSQL numeric types without precision

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#12040]
The JDBC backed `Meta` implementation produces a precision of `131089` for a table like this:

```sql
create table t (implicit_precision text)
```

This is what the JDBC driver produces, too:

```java
try (Statement s = connection.createStatement()) {
    try {
        s.executeUpdate("create table t (implicit_precision text)");
        try (ResultSet rs = connection.getMetaData().getColumns(null, "public", "t", "%")) {
            while (rs.next()) {
                System.out.println(rs.getInt(7));
            }
        }
    }
    finally {
        s.executeUpdate("drop table if exists t");
    }
}
``` 

See also: 

- https://github.com/pgjdbc/pgjdbc/discussions/2187
- https://github.com/pgjdbc/pgjdbc/issues/2188

We should ignore that precision.

**Comment by lukaseder** [comment]
The current state of discussion is that pgjdbc seems to fix this. I'm postponing any action here for now. It won't go into 3.15.0. If this gets fixed, it will be backported to a 3.15.x release.

**Comment by lukaseder** [comment]
https://github.com/pgjdbc/pgjdbc/issues/2188 was fixed by removing the "magic" `COLUMN_SIZE`" value `131089` for `NUMERIC` column types, meaning I won't fix this issue in jOOQ.
